src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeValueTranslationType.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\Type;
+
+use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Salvatore Pappalardo <salvatore.pappalardo82@gmail.com>
+ */
+class AttributeValueTranslationType extends AbstractResourceType
+{
+    /**
+     * @var string
+     */
+    protected $subjectName;
+
+    /**
+     * @param string $dataClass
+     * @param array $validationGroups
+     * @param string $subjectName
+     */
+    public function __construct($dataClass, array $validationGroups, $subjectName)
+    {
+        parent::__construct($dataClass, $validationGroups);
+        $this->subjectName = $subjectName;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $valueTranslationType = 'sylius_attribute_type_'.$options['valueTranslationType'];
+        $builder
+            ->add('value', $valueTranslationType, [
+                'label' => 'sylius.form.attribute.name',
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+        $resolver->setRequired('valueTranslationType');
```

Comment:
For options there is a convention to use `snake_case`, so `value_translation_type`.